src/SDKs/Azure.Core/data-plane/Azure.Core/Response.cs

```diff
@@ -8,38 +8,4 @@
 
 namespace Azure
 {
```

Comment:
May just be a weird diff... but it looks like the file still exists and just the contents are zapped....